src/main/java/Racing.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Scanner;
+
+public class Racing {
+    public void run(){
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분");
+        String [] players = (scanner.next()).split(",");
```

Comment:
코드 컨벤션을 지켜주셔야 합니다.
`Car[]`로 했다가 `String []`로 했다가 하면 읽기 힘들어질 수 있습니다.
```suggestion
        String[] players = (scanner.next()).split(",");
```